packages/streamline/src/components/buttons/button-icon/button-icon.tsx

```diff
@@ -84,7 +85,7 @@ export const ButtonIcon = ({
               borderRadius="round"
               opacity={activeColors.backgroundOpacity}
             />
-            {isLoading || isResolving ? (
+            {(isLoading || isResolving) && shouldDisplaySpinner ? (
```

Comment:
@hcourthias Are you sure we don't want to display spinner? Seems weird.

Reply:
For me it is weird to have a loader when going back. You can check the new Lukoff flow to reproduce it.

Reply:
https://github.com/GetLuko/streamline/assets/32702872/0ed015da-b74a-4e69-b66d-3b5bec1f4c9e

Reply:
I guess we always convert actions to Promise. If so, let's keep it as you did